Makefile.core.mk

```diff
@@ -225,7 +225,7 @@ test_status:
 	@scripts/test_status.sh
 
 update-gateway-version: tidy-go
-	@$(eval GATEWAY_VERSION := ${shell grep gateway-api go.mod | awk '{ print $$2 }' | cut -f1 -d'-'})
+	@$(eval GATEWAY_VERSION := ${shell grep gateway-api go.mod | awk '{ print $$2 }'})
```

Comment:
Possibly. We may decide that the pulling of the version from the go.mod is not correct, and for those occasions where it is updated that we should be update the arg.yaml manually. It would require a manual update when that occurs.